### DISCUSSION TITLE
AI: Superhero or super villain? 👾  [AI2768M]

### DISCUSSION THREAD

**Discussion by ghostinhershell** [community/community#74976] [Programming Help]
**Speakers:** 
- Scott Hanselman (VP Developer Community, Microsoft)

**Time:** Thursday, Nov 9, 3:45 PM - 4:25 PM PST

> Generative AI has captured the imagination of the public by storm, and it feels like there is a hot new large language model (LLM) being announced every other day. But as developers, we are trying to solve problems for people and make delightful experiences for them. Is AI here to help, or to hurt? In this demo packed session, Scott dives into the science behind LLMs, explains what they can and can’t do, and uncovers how we can use generative AI to make our lives better as developers and humans.

Thank you for joining the session! If you have any questions or feedback regarding what was presented, please feel free to add them in the comments and I'll/we'll do our best to answer.

To watch the recording, please click [`link tbd`](https://github.com/orgs/community/discussions/74976).👈

**Comment by ghostinhershell** [comment]
In a conversation about the ethics of AI, and how AI tools can be problematic depending on how they are used, here's a hilarious example of how AI tools will output exactly what you asked for. Our speaker asked the tool to share a 🌮 recipe with the sass of Benjamin Cumberbatch as Sherlock Holmes, and isn't in the mood to do so in the first place.

The results? 🌶️ insults included in the ingredients list 😆 
![The image displays the results of the speaker's query. The speaker asked the AI tool for a taco recipe. The following text in the image is highlighted: - Sour cream (to cool down the fiery flames of your culinary ignorance).](https://github.com/community/community/assets/14021469/2949e2cb-432e-4915-a73b-bd5201824b35)

**Comment by Alchemyst0x** [comment]
This was an excellent talk, very sobering. I definitely enjoyed that! I'll be sharing the recording with others now when it's available 😄 Quite possibly the best take I have heard on the subject of "hacking" a LLM.

**Comment by elviraIv** [comment]
I think AI  should be used for the benefit of humanity and with careful consideration of potential risks and consequences. For this reason it bears the possibility of being both a superhero or super villain.

**Reply by ravin-d-27** [reply]
Yes, if we introduce the rules and regulations which AI developers or the organization follow, then it would be the superhero of all time!
